lib/react_client.dart

```diff
@@ -448,11 +393,9 @@ _getValueFromDom(domElem) {
   }
 }
 
-/**
- * set value to props based on type of input.
- *
- * Specialy, it recognized chceckbox.
- */
+/// Set value to props based on type of input.
+///
+/// Specifically, it recognized checkbox.
```

Comment:
this line doesn't make sense...maybe `Processing checkbox 'checked' value is handled as a special case.`?